1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,5 +1,25 @@
-function setAlarm() {}
+function setAlarm() {
+  let timeDisplay = document.getElementById("timeRemaining");
+  let alarmValue = document.getElementById("alarmSet").value;
+  let extraZero = "";
 
+  if (alarmValue > 0 && parseInt(alarmValue) === eval(alarmValue)) {
+    if (alarmValue < 10) extraZero = "0";
+    const timeInterval = setInterval(function () {
+      updateTime();
+    }, 1000);
+  }
+  function updateTime() {
+    timeDisplay.innerHTML = `Time Remaining: 00:${extraZero}${alarmValue}`;
```

Comment:
I really like how you didn't add the extra zero to `alarmValue` directly 👍 because you've separated the display of the number from the storage of its value